src/app/CommandSender.h

```diff
@@ -217,6 +218,14 @@ class CommandSender final : public Messaging::ExchangeDelegate
     //
     CHIP_ERROR SendCommandRequest(const SessionHandle & session, System::Clock::Timeout timeout = kImMessageTimeout);
 
+    // Sends a queued up group command request to the target encapsulated by the secureSession handle.
+    //
+    // Upon completion of work regardless of whether it was successful or not, the OnDone
+    // callback will be invoked to indicate completion of work on this object and to indicate
+    // to the application that it can destroy and free this object.
```

Comment:
```suggestion
    // If this function is successful, it will invoke the OnDone callback before returning to indicate
    // to the application that it can destroy and free this object.
```
The "successful or not" makes no sense here, since there are no observable async errors and in the sync error cases OnDone is not in fact called.